documentation/synthetic_tokens/tutorials/mint_tokens_via_etherscan.md

```diff
@@ -1,22 +1,22 @@
-# Minting ETHBTC Synthetic Tokens via Etherscan
+# Minting Synthetic Tokens via Etherscan
 
-This article will walk you through minting ETHBTC tokens for the first time on Etherscan’s contract interaction GUI. If you already have a position (i.e. if you’ve already minted tokens, these steps may not apply to you).
+This article will walk you through minting tokens (like ETHBTC or xyCOMP) for the first time on Etherscan’s contract interaction GUI. If you already have a position (i.e. if you’ve already minted tokens, these steps may not apply to you). The screenshots and example numbers used in this tutorial reflect the ETHBTC token, but all steps in this article apply just the same to any other expiring multi-party token (like the xyCOMP).
 
-You can already mint tokens via the [CLI tool](./using_the_uma_sponsor_cli_tool.md), but you may prefer to interact with the contract more directly via Etherscan. All you’ll need is some DAI to use as collateral (50 DAI should be enough) and ETH to pay for transaction gas (0.05 ETH should be enough).
+You can already mint tokens via the [CLI tool](./using_the_uma_sponsor_cli_tool.md), but you may prefer to interact with the contract more directly via Etherscan. All you’ll need to get started is some DAI to use as collateral (50 DAI should be enough) and ETH to pay for transaction gas (0.05 ETH should be enough).
 
-## Before we begin: DAI approval
+## Before we begin: collateral approval
 
-The ETHBTC token minting contract needs approval to transfer DAI (the collateral currency) on your behalf. To do that, follow these instructions:
+The token minting contract needs approval to transfer the collateral currency (DAI in this case) on your behalf. To do that, follow these instructions:
 
 1. Go to the [Write Contract Tab](https://etherscan.io/address/0x6b175474e89094c44da98b954eedeac495271d0f#writeContract) on the [DAI](https://etherscan.io/token/0x6b175474e89094c44da98b954eedeac495271d0f) contract page.
 2. Search for the `approve` function.
-3. For the first argument, pass in the address of the ETHBTC token minting contract: `0x3f2d9edd9702909cf1f8c4237b7c4c5931f9c944`.
+3. For the first argument, pass in the address of the token minting contract (for ETHBTC on mainnet it is `0x3f2d9edd9702909cf1f8c4237b7c4c5931f9c944`).
```

Comment:
nit: thoughts on having an appendix or some other page where we keep a curated list of deployed EMP addresses?

Reply:
I think in the short-to-medium term we can manage a page like that in the docs, but I think it makes more sense when there are three or four EMPs.

In the long term, I'd like to see a frontend that lists all existing EMPs in the Registry.

Reply:
100% agreed. A dynamic dashboard would be the best here. SGTM -- we can put off creating a list until we have a few more EMPs.